src/SFA.DAS.AANHub.Application/Partners/Commands/PatchPartnerMember/PatchPartnerMemberCommandValidator.cs

```diff
@@ -0,0 +1,53 @@
+﻿using FluentValidation;
+using SFA.DAS.AANHub.Application.Common.Validators;
+using SFA.DAS.AANHub.Application.Common.Validators.RequestedByMemberId;
+using SFA.DAS.AANHub.Domain.Entities;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+
+namespace SFA.DAS.AANHub.Application.Partners.Commands.PatchPartnerMember
+{
+    public class PatchPartnerMemberCommandValidator : AbstractValidator<PatchPartnerMemberCommand>
+    {
+        private static readonly List<string> PatchFields =
+            new()
+            {
+                "Email",
+                "Name",
+                "Organisation"
+            };
+
+        public PatchPartnerMemberCommandValidator(IMembersReadRepository membersReadRepository)
+        {
+            Include(new PatchMemberCommandBaseValidator<Partner>(PatchFields));
```

Comment:
The next line of code does this. if we are doing this for the Create endpoint I guess that we need to for Patch as well

Reply:
We should return a `BadRequest` response if the member does not exist. So I think we will have to validate. 

In case of GET endpoint we do not need validate if member exist as we want to return `NotFound` response in that case rather than `BadRequest`. Hope this makes sense.